locations/spiders/alpha_bank_gr.py

```diff
@@ -0,0 +1,52 @@
+from scrapy import Request, Spider
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_GR, OpeningHours, day_range
+
+
+class AlphaBankGRSpider(Spider):
+    name = "alpha_bank_gr"
+    item_attributes = {"brand": "Alpha Bank", "brand_wikidata": "Q747394"}
+
+    def start_requests(self):
+        yield Request(url="https://www.alpha.gr/api/maps/MapLocations?bringPrivates=False&httproute=True")
+
+    def parse(self, response):
+        data = response.json()
+        for branch in data.get("Branches", []):
+            item = self.parse_poi(branch)
+            apply_category(Categories.BANK, item)
+
+            yield item
+
+        for atm in data.get("Atms", []):
+            item = self.parse_poi(atm)
+            apply_category(Categories.ATM, item)
+            item["branch"] = atm.get("BranchId")
```

Comment:
I suggest to use `name` instead of `BranchId` for `branch`:

```suggestion
            item["branch"] = item.pop("name")
```